### ISSUE TITLE
POSIX not available on Windows enviroments

### ISSUE THREAD

**Issue by angelside** [fwartner/laravel-installer#2]
Not working on Windows OS because; "POSIX not available on Windows enviroments"

```
- php-school/cli-menu 3.0.0 requires ext-posix * -> the requested PHP extension posix is missing from your system.
```

**Comment by nunomaduro** [comment]
I will try to find a solution for this. Ideally, we should configure
composer to ignore this requirement.

**Comment by fwartner** [comment]
@nunomaduro Any updates on this?

**Comment by nunomaduro** [comment]
This issue will be fixed on Laravel Zero 5.8. Sorry.
